oat-sa/tao-mobile-app
# Fix/tao 7496/line clamp profile in toolbar

Long names in the user profiles break the user interface.
In order to fix it I've changed a bit the DOM (to have a full flexbox row) and used the prefixed version of `line-clamp`. Hopefully, since the target is cordova (so webkit webviews) we can use those prefixed rules safely.

How to test : 
 - Contact me in private to get an account on the server to synchronize some user profiles
 - Clone the repository and checkout the branch
 - `npm install` 
 -  `npm start`
 - Open Chrome(ium) on `http://localhost:3000`
 - Login with a sync manager, and synchronize the test takers
 - Logout and  login as a test taker
 - Check the GUI with different resolutions and verify the long user name doesn't break the GUI.
